### DISCUSSION TITLE
Multiple Jira accounts

### DISCUSSION THREAD

**Discussion by oliveradk** [adyscorpius/logseq-jira#7] [Ideas]
My organization has an internal and external jira, with different projects on both, so it would be nice if the plugin allowed for multiple jira accounts

**Comment by adyscorpius** [comment]
Sounds like a good idea. 

Thinking loudly, how would you identify which account a ticket is from? 

I don't want to spam Jira with duplicate API calls.

**Reply by oliveradk** [reply]
Hmm, maybe different keyboard shortcuts for different accounts? (fwiw I haven't actually used the extension yet)

**Reply by dv-adit** [reply]
@oliveradk Hi, I was wondering if it's possible for you to do a test for two parallel accounts. I've committed a rudimentary solution that works but might not be the best UX. 

Would love to get your feedback on how you find it and if it works as expected.

**Comment by adyscorpius** [comment]
This is now supported.
